roles/ceph-common/handlers/inject-args.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: check the diff between old and new ceph.conf
+  command: "diff --changed-group-format='%>' --unchanged-group-format='' /etc/ceph/{{ cluster }}.conf /tmp/{{ cluster }}.conf"
+  register: diffcephconf
+
+- name: inject the configuration option
+  command: 
```

Comment:
Ah this PR is really WIP so it is not meant to work at all, just a way for me to save my work online just in case ;)